MapleServer2/Types/Mob.cs

```diff
@@ -1,19 +0,0 @@
-﻿using Maple2Storage.Types;
-
-namespace MapleServer2.Types
-{
-    public class Mob
-    {
-        public readonly int Id;
-        public short Rotation; // In degrees * 10
-        public CoordS Speed;
-        public byte Animation;
-        public PlayerStats stats;
-
-        public Mob(int id)
-        {
-            this.Id = id;
-            this.Animation = 255;
-        }
-    }
-}
```

Comment:
I do not understand why we are deleting things that we will eventually need, this makes zero sense.

Reply:
Again, the mob was made for testing purpose, and is not sure that is the same as NPC

Reply:
When I implemented mob spawning on my end, I used this existing implementation as the base, so this is code worth keeping and later improving once the emulator gets there